### PR TITLE
Papa 10 Deployment & Recovery Updates

### PR DESCRIPTION
**Pull request by snwhite**
Deployment and Recovery details provided by R. Travis from the ship:

> Here are the tables with mooring & glider deploy/recover info for Asset Mgt.  These have been checked and double-checked.
> **Please note one change for the GP03FLMB deploy csv: secondary controller/persistor swapped out during  Alaska burnin: 19/13998 removed and replaced with 10/11690 (so CGCON-SIOCON-13998 needs to be changed to CGCON-SIOCON-11690; and the new one is already in Eng Bulk Load).  Other than that...all as expected!

[Papa-10_QLR_tables_2023-05-22.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/11559260/Papa-10_QLR_tables_2023-05-22.pdf)

### CODE REVIEW

File: deployment/GP02HYPM_Deploy.csv [hunk 1]
```diff
@@ -105,31 +105,31 @@ SKQ202111S,,SKQ202208S,,GP02HYPM-WFP03-01-FLORDL000,8,1,2021-07-22T01:55:00,2022
...
+SKQ202308S,,,,GP02HYPM-WFP02-04-CTDPFL000,10,1,2023-05-17T19:16:00,,CGMGP-02HYPM-00010,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.07037,-144.80075,,2090,4218,,,
+SKQ202308S,,,,GP02HYPM-WFP02-05-VEL3DL000,10,1,2023-05-17T19:16:00,,CGMGP-02HYPM-00010,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.07037,-144.80075,,2090,4218,,,
+SKQ202308S,,,,GP02HYPM-WFP03-00-WFPENG000,10,1,2023-05-17T19:16:00,,CGMGP-02HYPM-00010,CGVEH-GWFPGX-10407,CGCON-GWFPGX-10407,50.07037,-144.80075,,4063,4219,,,
+SKQ202308S,,,,GP02HYPM-WFP03-01-FLORDL000,10,1,2023-05-17T19:16:00,,CGMGP-02HYPM-00010,CGVEH-GWFPGX-10407,CGINS-FLORDL-03279,50.07037,-144.80075,,4063,4218,,,
+SKQ202308S,,,,GP02HYPM-WFP03-03-DOSTAL000,10,1,2023-05-17T19:16:00,,CGMGP-02HYPM-00010,CGVEH-GWFPGX-10407,CGINS-DOSTAL-01479,50.07037,-144.80075,,4063,4218,,,
+SKQ202308S,,,,GP02HYPM-WFP03-04-CTDPFL000,10,1,2023-05-17T19:16:00,,CGMGP-02HYPM-00010,CGVEH-GWFPGX-10407,CGINS-CTDPFL-00130,50.07037,-144.80075,,4063,4218,,,
+SKQ202308S,,,,GP02HYPM-WFP03-05-VEL3DL000,10,1,2023-05-17T19:16:00,,CGMGP-02HYPM-00010,CGVEH-GWFPGX-10407,CGINS-VEL3DL-01024,50.07037,-144.80075,,4063,4218,,,
```
Comment (sbeaulieu) [inline deployment/GP02HYPM_Deploy.csv:135]: looks good

File: deployment/GP03FLMA_Deploy.csv [hunk 2]
```diff
@@ -143,39 +143,39 @@ SKQ202111S,,SKQ202208S,,GP03FLMA-RIS01-00-SIOENG000,8,1,2021-07-23T19:43:00,2022
...
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG048,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-10253,49.978,-144.246,,500,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOH049,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOH-12227,49.978,-144.246,,750,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOH050,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOH-12619,49.978,-144.246,,1000,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOH051,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOH-11703,49.978,-144.246,,1500,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-00-SIOENG000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGCON-SIOCON-13997,49.978,-144.246,,30,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-03-DOSTAD000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-DOSTAD-00503,49.978,-144.246,,30,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-04-PHSENF000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-PHSENF-P0080,49.978,-144.246,,30,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-05-FLORTD000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-FLORTD-01116,49.978,-144.246,,30,4127,,,
```
Comment (sbeaulieu) [inline deployment/GP03FLMA_Deploy.csv:181]: Looks good. I looked at previous deployments for expectations for recovery at anchor vs midwater.

File: deployment/GP03FLMB_Deploy.csv [hunk 3]
```diff
@@ -143,39 +143,39 @@ SKQ202111S,,SKQ202208S,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-24T19:32:00,2022
...
+SKQ202308S,,,,GP03FLMB-RIM01-02-CTDMOG068,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGINS-CTDMOG-13597,50.3315,-144.3976,,500,4145,,,
+SKQ202308S,,,,GP03FLMB-RIM01-02-CTDMOH069,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGINS-CTDMOH-13455,50.3315,-144.3976,,750,4145,,,
+SKQ202308S,,,,GP03FLMB-RIM01-02-CTDMOH070,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGINS-CTDMOH-10227,50.3315,-144.3976,,1000,4145,,,
+SKQ202308S,,,,GP03FLMB-RIM01-02-CTDMOH071,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGINS-CTDMOH-13449,50.3315,-144.3976,,1500,4145,,,
+SKQ202308S,,,,GP03FLMB-RIS01-00-SIOENG000,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGCON-SIOCON-11690,50.3315,-144.3976,,30,4145,,,
+SKQ202308S,,,,GP03FLMB-RIS01-03-DOSTAD000,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGINS-DOSTAD-00515,50.3315,-144.3976,,30,4145,,,
+SKQ202308S,,,,GP03FLMB-RIS01-04-PHSENF000,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGINS-PHSENF-P0144,50.3315,-144.3976,,30,4145,,,
+SKQ202308S,,,,GP03FLMB-RIS01-05-FLORTD000,10,1,2023-05-19T19:40:00,,CGMGP-03FLMB-00010,,CGINS-FLORTD-01217,50.3315,-144.3976,,30,4145,,,
```
Comment (sbeaulieu) [inline deployment/GP03FLMB_Deploy.csv:181]: Looks good

File: deployment/GP05MOAS-PG515_Deploy.csv [hunk 4]
```diff
@@ -6,3 +6,10 @@ TN323,,TY-1402,,GP05MOAS-PG515-03-FLORTM000,2,1,2015-06-02T09:55:00,2016-02-17T2
...
+SKQ202308S,,,,GP05MOAS-PG515-00-ENG000000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGCON-GLDRGP-00515,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-01-CTDGVM000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-02-DOSTAM000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00192,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-03-FLORTM000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTM-03501,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-04-FLORTO000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-05-NUTNRM000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-NUTNRM-00760,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-06-PARADM000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50213,50.46368,-144.85315,,200,200,,,
```
Comment (snwhite) [inline deployment/GP05MOAS-PG515_Deploy.csv:15]: Looks good, but DOSTA doesn't have "top mount" note.

File: deployment/GP05MOAS-G0899_Deploy.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+SKQ202308S,,SKQ202308S,,GP05MOAS-G0899-00-ENG000000,1,1,2023-05-16T15:46:00,2023-05-18T03:20:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGCON-GLDROO-00899,50.47069,-144.83979,,1000,1000,,,
+SKQ202308S,,SKQ202308S,,GP05MOAS-G0899-01-FLORDM000,1,1,2023-05-16T15:46:00,2023-05-18T03:20:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-FLORDM-06990,50.47069,-144.83979,,1000,1000,,,
+SKQ202308S,,SKQ202308S,,GP05MOAS-G0899-02-DOSTAM000,1,1,2023-05-16T15:46:00,2023-05-18T03:20:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-DOSTAM-00323,50.47069,-144.83979,,1000,1000,top mount,,
+SKQ202308S,,SKQ202308S,,GP05MOAS-G0899-04-CTDGVM000,1,1,2023-05-16T15:46:00,2023-05-18T03:20:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00899,CGINS-CTDGVM-09686,50.47069,-144.83979,,1000,1000,,,
```
Comment (snwhite) [inline deployment/GP05MOAS-G0899_Deploy.csv:5]: Looks good.

File: deployment/GP03FLMB_Deploy.csv (hunk 3 above)
Comment (cdobs) [reply deployment/GP03FLMB_Deploy.csv:181]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00760__20230514.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
...273.71, 274.52, 275.32, 276.13, 276.94, 277.75, 278.56, 279.37, 280.18, 280.99, 281.8, 282.61, 283.42, 284.23, 285.04, 285.85, 286.65, 287.46, 288.27, 289.08, 289.89, 290.7, 291.51, 292.32, 293.13, 293.94, 294.75, 295.56, 296.37, 297.18, 297.99, 298.8, 299.61, 300.42, 301.23, 302.04, 302.84, 303.65, 304.46, 305.27, 306.08, 306.89, 307.7, 308.51, 309.32, 310.13, 310.94, 311.75, 312.56, 313.37, 314.17, 314.98, 315.79, 316.6, 317.41, 318.22, 319.03, 319.84, 320.64, 321.45, 322.26, 323.07, 323.88, 324.69, 325.49, 326.3, 327.11, 327.92, 328.73, 329.53, 330.34, 331.15, 331.96, 332.76, 333.57, 334.38, 335.19, 335.99, 336.8, 337.61, 338.41, 339.22, 340.03, 340.83, 341.64, 342.45, 343.25, 344.06, 344.86, 345.67, 346.48, 347.28, 348.09, 348.89, 349.7, 350.5, 351.31, 352.11, 352.92, 353.72, 354.53, 355.33, 356.13, 356.94, 357.74, 358.55, 359.35, 360.15, 360.96, 361.76, 362.56, 363.36, 364.17, 364.97, 365.77, 366.57, 367.38, 368.18, 368.98, 369.78, 370.58, 371.38, 372.18, 372.98, 373.79, 374.59, 375.39, 376.19, 376.99, 377.79, 378.58, 379.38, 380.18, 380.98, 381.78, 382.58, 383.38, 384.18, 384.97, 385.77, 386.57, 387.37, 388.16, 388.96, 389.76, 390.55, 391.35, 392.14, 392.94, 393.73, 394.53]",
```
Comment (snwhite) [inline calibration/NUTNRM/CGINS-NUTNRM-00760__20230514.csv:8]: Looks good.

**Review by snwhite — COMMENTED** [review]
@cdobs Glider 515 doesn't have the "top mount" note.

**Review by cdobs — APPROVED** [review]
looks good

File: deployment/GP05MOAS-PG515_Deploy.csv [hunk 7]
```diff
@@ -6,3 +6,10 @@ TN323,,TY-1402,,GP05MOAS-PG515-03-FLORTM000,2,1,2015-06-02T09:55:00,2016-02-17T2
 TN323,,TY-1402,,GP05MOAS-PG515-04-FLORTO000,2,1,2015-06-02T09:55:00,2016-02-17T20:04:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,50.16765,-144.5446,,200,200,Actual deployment port # was 03,,
 #TN323,,TY-1402,,GP05MOAS-PG515-05-NUTNRM000,2,1,2015-06-02T09:55:00,2016-02-17T20:04:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,,50.16765,-144.5446,,200,200,not deployed,,
 TN323,,TY-1402,,GP05MOAS-PG515-06-PARADM000,2,1,2015-06-02T09:55:00,2016-02-17T20:04:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50185,50.16765,-144.5446,,200,200,Actual deployment port # was 04,,
+SKQ202308S,,,,GP05MOAS-PG515-00-ENG000000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGCON-GLDRGP-00515,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-01-CTDGVM000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,,,GP05MOAS-PG515-02-DOSTAM000,5,1,2023-05-16T15:12:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00192,50.46368,-144.85315,,200,200,top mount,,
```
Comment (snwhite) [inline deployment/GP05MOAS-PG515_Deploy.csv:11]: Looks good.

File: deployment/GP02HYPM_Deploy.csv (hunk 1 above)
Comment (sauuyer) [reply deployment/GP02HYPM_Deploy.csv:135]: checked. looks good

File: deployment/GP03FLMA_Deploy.csv (hunk 2 above)
Comment (sauuyer) [reply deployment/GP03FLMA_Deploy.csv:181]: checked. looks good

File: deployment/GP03FLMB_Deploy.csv (hunk 3 above)
Comment (sauuyer) [reply deployment/GP03FLMB_Deploy.csv:181]: checked. looks good

File: deployment/GP05MOAS-PG515_Deploy.csv (hunk 4 above)
Comment (sauuyer) [reply deployment/GP05MOAS-PG515_Deploy.csv:15]: checked. looks good.

File: deployment/GP05MOAS-PG515_Deploy.csv (hunk 7 above)
Comment (sauuyer) [reply deployment/GP05MOAS-PG515_Deploy.csv:11]: checked looks good.

**Review by sauuyer — COMMENTED** [review]
all reviewed and they look good

**Review by sauuyer — APPROVED** [review]

**Pull request merged**